### ISSUE TITLE
Add dependencies for script development environment docs

### ISSUE THREAD

**Issue by wburnett** [pixie-io/docs.px.dev#15]
**Is your feature request related to a problem? Please describe.**
It looks like Bazel 4.0+ is a required dependency for the development environment. While the script logging explains that you need it once you run `make dev`, it would be nice to set up beforehand. The default suggested version on [Bazel's MacOS install documentation](https://docs.bazel.build/versions/master/install-os-x.html) is 3.2, so you need to reinstall/upgrade to 4.0 after running the make command

**Describe the solution you'd like.**
Add dependencies and versions to the [Script Dev Environment Tutorial page]( https://docs.pixielabs.ai/tutorials/script-dev-environment/#setup-the-script-dev-environment), somewhere around steps 1-3

**Additional context.**
TL;DR: low priority quality-of-life fix, this isn't blocking anything.

**Comment by htroisi** [comment]
Bazel was removed as a dependency in https://github.com/pixie-labs/pixie/commit/2f1b62ac7dcc451dd5c3bf8debbc9eafbd7fd5af.
